data/data/alibabacloud/vpc/sg-worker.tf

```diff
@@ -0,0 +1,272 @@
+resource "alicloud_security_group" "sg_worker" {
+  name        = "${local.prefix}-sg_worker"
+  description = local.description
+  vpc_id      = alicloud_vpc.vpc.id
+  tags = merge(
+    {
+      "Name" = "${local.prefix}-sg-worker"
+    },
+    var.tags,
+  )
+}
+
+resource "alicloud_security_group_rule" "sg_rule_worker_ingress_mcs" {
+  type              = "ingress"
+  ip_protocol       = "tcp"
+  policy            = "accept"
+  port_range        = "22623/22623"
+  security_group_id = alicloud_security_group.sg_worker.id
+  cidr_ip           = var.vpc_cidr_block
+}
+
+resource "alicloud_security_group_rule" "sg_rule_worker_ingress_icmp" {
+  type              = "ingress"
+  ip_protocol       = "icmp"
+  policy            = "accept"
+  port_range        = "-1/-1"
+  security_group_id = alicloud_security_group.sg_worker.id
+  cidr_ip           = var.vpc_cidr_block
+}
+
+resource "alicloud_security_group_rule" "sg_rule_worker_ingress_ssh" {
+  type              = "ingress"
+  ip_protocol       = "tcp"
+  policy            = "accept"
+  port_range        = "22/22"
+  security_group_id = alicloud_security_group.sg_worker.id
+  cidr_ip           = var.vpc_cidr_block
+}
+
+resource "alicloud_security_group_rule" "sg_rule_worker_ingress_https" {
+  type              = "ingress"
+  ip_protocol       = "tcp"
+  policy            = "accept"
+  port_range        = "6443/6443"
```

Comment:
I do not think port `6443` is required on the nodes.

Reply:
Finish updated